src/entity/Unit.ts

```diff
@@ -159,7 +157,7 @@ export function create(
     }
     const unit: IUnit = Object.assign({
       type: 'unit',
-      id: prediction ? ++lastPredictionUnitId : ++underworld.lastUnitId,
+      id: ++underworld.lastUnitId,
```

Comment:
I think this needs to remain different for prediction.  The prediction unit creation runs a LOT (and not the same number of times client to client), we want to make sure the clients' lastUnitId is always the same and so it needs to be different from prediction ids